app/views/users/_form.html.erb

```diff
@@ -1,6 +1,5 @@
 <div style="padding:10px;">
-<%= form_for @user, :url => {:controller => 'users', :action => @action}, :html => {:class => "row"} do |f| %>
-
+ <%= form_for :user, :as => :user, :url => :users, :html => {:class => "form col-md-6 offset-3"} do |f| %>
```

Comment:
Try 
<%= form_for @user, :url => {:controller => 'users', :action => 'new'}, :html => {:class => "row"} do |f| %>

Reply:
With these changes, we'll get:
![image](https://user-images.githubusercontent.com/22998430/49690615-f2b09d00-fb01-11e8-96cf-37f9b6b6a173.png)